### ISSUE TITLE
Detail View: Overlapping Close Button with Text or Info Buttons

### ISSUE THREAD

**Issue by cgg-cmu** [cmu-delphi/www-covidcast#597]
**Actual Behavior:**

When I have the detail view open for a signal, the close button and the info button can overlap if the subtitle is very long

**Steps to Reproduce the Bug:**

1. Go to https://covidcast.cmu.edu/?sensor=doctor-visits-smoothed_adj_cli&level=county&date=20201024&signalType=value&encoding=color&mode=overview&region=42003
2. (That is detail view of COVID-LIKE SYMPTOMS IN COMMUNITY )
3. Resize window to width between 1395px  to 1355px, in and around this range the buttons or button and text overlap

**Expected behavior**

The close button and info buttons should not overlap.

**Screenshots**

<img width="1356" alt="Screen Shot 2020-10-28 at 10 45 29 AM" src="https://user-images.githubusercontent.com/60197405/97454878-9b8e5e00-190d-11eb-950d-3e0d4e50441f.png">

**Rating Scale (1 is minor, 7 is severe):**

2

**Context**

- Browser: Mac Os Chrome

**Comment by dlaliberte** [comment]
This should be fixed with my change that moves the chart title/subtitle into the chart, and moves the buttons to the top right corner.   See https://github.com/cmu-delphi/www-covidcast/pull/573.
